src/HtmlGenerator/Utilities/Markup.cs

```diff
@@ -236,6 +236,7 @@ public static void WriteLinkPanel(Action<string> writeLine, params string[] rows
                 writeLine(row);
             }
 
+            writeLine(string.Format("<tr><td>File indexed on: {0}</td></tr>", DateTime.Now.ToString()));

```

Comment:
Also I'm thinking it's better to have the "File indexed on" at the end of the source code, not on the bottom panel. This way we don't waste valuable space and people who want to know when the file was indexed can just scroll to the bottom.